client/src/flaws/index.tsx

```diff
@@ -101,7 +101,7 @@ function withoutDefaultFilters(filters: Filters): Partial<Filters> {
  * NOTE: This only changes the given filters, and doesn't reset what is missing
  */
 function useFiltersURL(): [Filters, (filters: Partial<Filters>) => void] {
-  const [searchParams, setSearchParams] = (useSearchParams as any)();
+  const [searchParams, setSearchParams] = useSearchParams();
```

Comment:
This fixes a bug in an old react-router [that Gregor had to work around](https://github.com/mdn/yari/pull/602#issue-418167215).